src/NavConfig.jsonc

```diff
@@ -66,7 +66,10 @@
           "enabledByDefault": false,
           "visible": true
         }
-      ]
+      ],
+      "openPage": {
+        "key": "DevHome.Customization.ViewModels.MainPageViewModel"

```

Comment:
Just thinking out loud for the future. If we implement a protocol processor in Dev Home to help us navigate between pages and activate Dev Home with a protocol to perform certain actions (e.g. something like ms-devhome://{page}?{params}). We may be able to use that here as well and maintain a single implementation. Or both can also co-exist as long as the engine processing the two interfaces is the same.

#NoActionRequired